bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "remotestorage-modules",
+  "description": "remotestorage modules collection",
+  "keywords": "remotestorage modules collection",
+  "main": "src/*.js",
+  "ignore": [
+  ],
+  "version": "0.0.1",
+  "private": false,
+  "author": "Niklas Cathor <nilclass@riseup.net>",
```

Comment:
Bower doesn't even define an author field, I just left it in as it probably wouldn't really do harm. The information in package.json even used incorrect syntax -- I've fixed that now. According to [the docs](https://docs.npmjs.com/files/package.json), the author is a single person (i.e. the person who started the project), and you can add a contributors array for other people. I've declined doing that for now as that would require looking up everybody's email address and having them consent with publishing it here.